setup.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/sh -e
+
+OS=$(uname -s)
+ARCH=$(uname -m)
+LATEST_RELEASE_VERSION="0.9.1"
```

Comment:
Let's not hardcode this in the file, please. You can just pull `latest`. Otherwise we'll need to update it every single release

Reply:
I wasn't able to find any binaries with the latest tag for eg: ` pg_search_15-latest-1PARADEDB.el9.x86_64.rpm `.
So we'll need to specify this version.

Reply:
You can use the URL https://github.com/paradedb/paradedb/releases/latest which will autoupdate to the latest release. After that you can fetch the binary via a glob pattern without specifying 0.9.1

Reply:
Done! Instead of going with the glob url I fetched the latest version using the github api, and its working now!

Reply:
Thats sweet! Great work :) I’m moving apartment today so I might be slow but I will review this soon.